File: 240808/Next Level/next-level.cpp
#include <iostream>
#include <string>
using namespace std;

class My {
    public:
        string id;
        int level;

        My(string id = "codetree", int level = 10) {
            this -> id = id;
            this -> level = level;
        }

};

int main() {

    string c;
    int s;
    cin >> c >> s;

    My a = My("codetree", 10);
    My b = My(c,s);

    cout << "user " << a.id << " lv " << a.level << '\n';
    cout << "user " << b.id << " lv " << b.level;



    return 0;
}

File: 241019/Next Level/next-level.py
class next_level :
    def __init__(self, id=0, level=0) :
        self.id = id
        self.level = level

st1 = next_level("codetree", 10)
st2 = next_level()



st2.id, st2.level = tuple(input().split())

print(f"user {st1.id} lv {st1.level}")
print(f"user {st2.id} lv {st2.level}")